src/components/Dropdown/Dropdown.stories.js

```diff
@@ -103,160 +93,17 @@ WithObjectOptions.args = {
 
 export const WithStringOptions = Template.bind({});
 WithStringOptions.args = {
-  id: 'pokemon',
-  label: 'Choose a Pokemon',
+  id: 'fav-pet',
+  label: 'Favorite pet',
+  placeholder: 'Choose a pet',
   options: [
-    'Bulbasaur',
```

Comment:
I liked pokemons :(

Reply:
ah sorry - how about we keep them but only have 10 in the list? it felt a bit much to have that long list of unpronounceable names